draft-packages/avatar/KaizenDraft/Avatar/Avatar.tsx

```diff
@@ -98,32 +126,37 @@ export const Avatar = ({
       {avatarState !== "none" && (
         <img
           ref={image}
-          className={styles.avatarImage}
+          className={cx(styles.avatarImage, {
+            [styles.companyAvatarImage]: isCompany,
+          })}
           src={avatarSrc}
           onError={onImageFailure}
           onLoad={onImageSuccess}
-          alt={fullName}
+          alt={fullName || ""}
         />
       )}
-      {(avatarState === "none" || avatarState === "error") &&
-        (disableInitials || initials === "" ? (
-          fallbackIcon
-        ) : (
-          <span
-            className={cx(styles.initials, {
-              [styles.longName]: isLongName,
-            })}
-          >
-            {isLongName ? (
-              // Only called if 3 or more initials, fits text width for long names
-              <Textfit mode="single" max={getMaxFontSizePixels(size)}>
-                {initials}
-              </Textfit>
-            ) : (
-              getInitials(fullName, size === "small")
-            )}
-          </span>
-        ))}
+      {!isCompany
+        ? (avatarState === "none" || avatarState === "error") &&
+          (disableInitials || initials === "" ? (
```

Comment:
This mega conditional was hard to read and understand before, and it's now become a bit harder 😅, you might want to consider extracting it out into a function.

Reply:
I've split the initials into its own render function to make this a little less gnarly to read